### PR TITLE
Add locator.tap

### LINKED ISSUE #356: locator.tap([options])
The [`locator.tap([options])`](https://playwright.dev/docs/api/class-locator#locator-tap) method taps on the element that matches the locator's selector (with strict mode).

### PR DESCRIPTION
**Pull request by inancgumus**
Closes #356.

### CODE REVIEW

File: common/frame.go [hunk 1]
```diff
@@ -1594,27 +1594,30 @@ func (f *Frame) SetInputFiles(selector string, files goja.Value, opts goja.Value
 	// TODO: needs slowMo
 }
 
+// Tap the first element matches the selector.
```
Comment (imiric) [inline common/frame.go:1597]:
```suggestion
// Tap the first element that matches the selector.
```

File: tests/static/locators.html [hunk 2]
```diff
@@ -32,7 +32,10 @@
         );
         document.querySelector('#inputText').addEventListener(
             'mousemove', e => { result = true; }, false
-        );        
+        );
+        document.addEventListener(
+            'touchstart', e => { result = true; }, false
```
Comment (imiric) [inline tests/static/locators.html:37]:
```suggestion
            'touchstart', e => { window.result = true; }, false
```

**Review by imiric — CHANGES_REQUESTED** [review]
LGTM, just minor things.

Also, I think there's a bug here:
https://github.com/grafana/xk6-browser/blob/3ea1fdcfe4526151c7a4e126358001de9249f17b/common/element_handle.go#L593-L595

Shouldn't this be `tap(p.X, p.Y)`? Can you fix this in another PR?

File: tests/static/locators.html (hunk 2 above)
Comment (inancgumus) [reply tests/static/locators.html]: Thanks.

**Comment by inancgumus** [comment]
> Also, I think there's a bug here:
> 
> https://github.com/grafana/xk6-browser/blob/3ea1fdcfe4526151c7a4e126358001de9249f17b/common/element_handle.go#L593-L595
> 
> Shouldn't this be `tap(p.X, p.Y)`? Can you fix this in another PR?

Nice catch! Thanks. I fixed it in this PR as the tap test needs the fix to work.

**Review by ankur22 — APPROVED** [review]

**Pull request merged**